doc/website/getting-started/examples/singleprocess.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How to communicate in a single process between threads
```

Comment:
I do not think threads need to be mentioned here (besides, they are technically not necessary but without there might be little point except for using the same interface for every communication).

Reply:
IMHO I keep the threads in there, to put extra emphasis on them